crates/pipeline-manager/src/runner/error.rs

```diff
@@ -250,6 +262,16 @@ impl Display for RunnerError {
                     timeout.as_secs()
                 }
             }
+            Self::AutomatonSuspendingComputeTimeout { timeout } => {
+                writedoc! {
+                    f,
+                    "
+                    Pipeline could not have its compute resources suspended within the timeout of {}s.
+                    A possible reason is that it takes too long for Kubernetes to scale down.
```

Comment:
"The pipeline could not suspend its compute resources within the timeout of {}s. This can happen if Kubernetes takes too long to scale down these resources."